src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,34 +1,77 @@
-export const TodoFilter = () => (
-  <form className="field has-addons">
-    <p className="control">
-      <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
-        </select>
-      </span>
-    </p>
+import { FilterType } from '../../types/Filter';
 
-    <p className="control is-expanded has-icons-left has-icons-right">
-      <input
-        data-cy="searchInput"
-        type="text"
-        className="input"
-        placeholder="Search..."
-      />
-      <span className="icon is-left">
-        <i className="fas fa-magnifying-glass" />
-      </span>
+interface Props {
+  setFilterBy: (value: FilterType) => void;
+  filterBy: FilterType,
+  setQuery: (value: string) => void;
+  query: string;
+}
+export const TodoFilter: React.FC<Props> = ({
+  setFilterBy,
+  filterBy,
+  setQuery,
+  query,
+}) => {
+  const handlerChangeSelect = (event: React.ChangeEvent<HTMLSelectElement>) => {
```

Comment:
You write almost the same code 3 times. Try to get new sort type this way 👇 

const newFilterType = event.target.value as FilterType;